spec/scss_lint/linter/url_format_spec.rb

```diff
@@ -31,6 +31,12 @@
 
       it { should_not report_lint }
     end
+
+    context 'when URL contains a variable' do
+      let(:url) { '${' }
```

Comment:
Well the idea is to not fail if the url has an sort of variable match.  I mean I can change it so it allows `#{` or `{{` but isn't the idea that leaving it open like that it doesn't matter?

or should it be `{ '${url}' }?